plugins/modules/ios_vrf_global.py

```diff
@@ -0,0 +1,677 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2024 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for ios_vrf_global
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ios_vrf_global
+short_description: Resource module to configure global VRF definitions.
+description: This module provides declarative management of VRF definitions on Cisco IOS.
+version_added: 8.0.0
+author: Ruchi Pakhle (@Ruchip16)
+notes:
+  - Tested against Cisco IOS-XE version 17.3 on CML.
+  - This module works with connection C(network_cli).
+    See U(https://docs.ansible.com/ansible/latest/network/user_guide/platform_ios.html)
```

Comment:
add note that yaml callback is used to put examples
https://github.com/ansible-collections/cisco.ios/blob/37d99e153e3190612aa380fdc8582e9f6f091931/plugins/modules/ios_bgp_global.py#L28